src/main/java/com/google/api/codegen/ApiaryConfig.java

```diff
@@ -102,15 +102,64 @@
   private final Table<Type, String, Field> fields = HashBasedTable.<Type, String, Field>create();
 
   /*
-   * The service canonical name, or name if no canonical name
+   * The service canonical name, or name if no canonical name.
    */
   private String serviceCanonicalName;
 
   /*
-   * The service version string
+   * The service version string.
    */
   private String serviceVersion;
 
+  /*
+   * The auth instructions URL.
+   */
+  private String authInstructionsUrl;
```

Comment:
s/Url/Urls/, and pluralize the comment above, expanding it to clarify that it's a flattened list of per-language URLs.